_data/version2-english/glossary.yml

```diff
@@ -68,7 +68,19 @@ Non-Finalized Market: <i>under construction</i>
 Open Interest: <i>under construction</i>
 Open Reporting Phase: <i>under construction</i>
 Outcome: <i>under construction</i>
-Parent Universe: <i>under construction</i>
+Parent Universe: |-
+  A parent universe is a <a href="#Universe">universe</a> that has spawned <a href="#Child_Universe">child universes</a> because a <a href="#Fork">fork</a> had occurred in the parent universe.
+  <br>
+  <br>
+  For 60 days after a fork starts (the 60 days is called <a href="#Forking_Period">forking period</a>), <a href="#Reputation_Token">REP</a> in the parent universe can be migrated to a child universe. But after the forking period, you cannot migrate your REP to any child universe.
+  <br>
+  <br>
+  After a fork starts, the parent universe is called <a href="#Locked_Universe">locked universe</a> because there are some restrictions:
+  Users cannot create new <a href="#Market">markets</a> and cannot stake their REP on any markets, since REP in the parent universe must be migrated to one of the child universes during the forking period.
+  Therefore markets in the parent universe are never <a href="#Finalized_Market">finalized</a> after a fork starts.
+  <br>
+  But users can trade and <a href="#Settlement">settle</a> their <a href="#Share">shares</a> on markets, and purchase and redeem <a href="#Participation_Token">participation tokens</a> in the parent universe even after a fork starts.
```

Comment:
```suggestion
  Users can trade and <a href="#Settlement">settle</a> their <a href="#Share">shares</a> on markets, and purchase and redeem <a href="#Participation_Token">participation tokens</a> in the parent universe even after a fork starts.
```

Reply:
Are you sure about being able to purchase participation tokens?  I thought that was disabled since all REP should be migrating as part of the fork?

Reply:
I'm going to try to verify that.

Reply:
Oh, you are right. Due to [this requirement](https://github.com/AugurProject/augur/blob/853e5bfb2fa31b9cf694b9c467c68f687fc17bb2/packages/augur-core/src/contracts/reporting/DisputeWindow.sol#L114), you cannot purchase participation tokens in the universe where a fork occurs.
I need to fix some contents in augur.guide. Thank you for letting me know 🌝